src/webview/serverless-function/app/createFunction.tsx

```diff
@@ -0,0 +1,332 @@
+                        error={functionData.error}
+                        onChange={(e) => this.validateName(e.target.value)}
+                        id='function-name'
+                        placeholder='Function Name *'
+                        sx={{
+                            input: {
+                                color: 'var(--vscode-settings-textInputForeground)',
+                                height: '5px !important',
+                                '&::placeholder': {
+                                    color: 'var(--vscode-settings-textInputForeground) !important',
+                                    opacity: '1 !important'
+                                }
+                            }
+                        }}
+                        helperText={functionData.helpText}
+                    />
+                </FormControl>
+                <FormControl sx={{ margin: '2rem 0 0 2rem', width: 'auto' }}>
+                    <Autocomplete
+                        value={imageData.name}
+                        disabled={functionData.error || functionData.name.length === 0}
+                        onChange={(_event, newValue: string) => {
+                            if (newValue) {
+                                const value = newValue.replace('Add', '').trim();
+                                if (!imageRegex.test(value)) {
+                                    this.setState({
+                                        imageData: {
+                                            name: '',
+                                            error: true,
+                                            helpText: 'Provide full image name in the form [registry]/[namespace]/[name]:[tag]'
+                                        }
+                                    });
+                                } else {
+                                    if (!images.includes(value)) {
+                                        images.push(value);
+                                    }
+                                    this.setState({
+                                        imageData: {
+                                            name: value,
+                                            error: false,
+                                            helpText: ''
+                                        }
+                                    });
+                                }
+                            }
+                        }}
+                        filterOptions={(options, params) => {
+                            const filtered = filter(options, params);
+
+                            const { inputValue } = params;
+                            // Suggest the creation of a new value
+                            const isExisting = options.some((option) => inputValue === option);
+                            if (inputValue !== '' && !isExisting) {
+                                filtered.push(`Add ${inputValue}`);
```

Comment:
If user can `Add` a new image name, there should be a provision to remove that added name also. Right now we can only add, but cannot remove that.

If possible, add a delete icon at the end of the select option in the dropdown for the specific added build image.

Reply:
<img width="509" alt="Screenshot 2023-07-19 at 2 36 50 AM" src="https://github.com/redhat-developer/vscode-openshift-tools/assets/3133356/b72dfe45-b48f-4694-b0ef-753f0712349e">

For example, delete icon at the right of the last row (as that was added)

Reply:
The delete icon was not possible on the `options`